Cargo.toml

```diff
@@ -35,5 +36,13 @@ chrono = "0.4"
 log = "0.4"
 fern = "0.6"
 
+widgets = { path = "crates/widgets" }
```

Comment:
I think it's fine to have it in `crates/widgets` - maybe too specific to break out of ajour, and too generic to stay inside the `src` folder. I think this is a good solution.